components/utils/parameters.ts

```diff
@@ -38,3 +38,21 @@ export const sortOptions = (parameter: Parameter, optionsToSort: string[]) => {
     return getIndexOfOption(parameter.options, a) - getIndexOfOption(parameter.options, b);
   });
 };
+
+// Given a numeric parameter where the RangeData is dependent on another parameter's value,
+// and the current values for each parameter (which could be data in a form, or a current scenario),
+// retrieve the RangeData (e.g. min, max and default).
+export const getRangeForDependentParam = (
```

Comment:
I don't really understand how the current parameter value influences what the correct range is. Is this a bootstrapping/reactvitiy issue?